ateliermecanique-ws/src/main/java/com/champlain/ateliermecaniquews/appointmentmanagementsubdomain/datalayer/Appointment.java

```diff
@@ -21,7 +19,7 @@ public class Appointment {
         @Embedded
         private AppointmentIdentifier appointmentIdentifier;
 
-        @Column(name = "customer_id")
+        @Column(name = "user_id")

```

Comment:
Can you explain why you changed customer_id to user_id?

Reply:
Customers are now users with the role Customer, therefore I had to change every customer_id to user_id